demo/screengrabs/build-all.sh

```
@@ -2,61 +2,78 @@
+		SED="sed -i"    # Linux does not require ''
+fi
+
 function record() {
-		local demo=$1
+    local demo=$1
     local command=$2
-		local extension="${command##*.}" # if any...
-		local demo_path=../../examples/$demo
+    local extension="${command##*.}" # if any...
+    local demo_path=../../examples/$demo
     local output_base_file=artifacts/$(echo "$command" | sed -E 's/ -/-/g' | sed -E 's/ +/-/g' | sed 's/---/--/g' | sed 's/scripts\///' | sed 's/\.sh$//')
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)